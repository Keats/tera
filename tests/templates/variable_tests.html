{% if dhfpuoashbjk is defined %}
  <h1>Oh no!</h1>
{% else %}
  <h1>Hello!</h1>
{% endif %}

{% if product is defined %}
  <h1>Hello again!</h1>
{% else %}
  <h1>Oh no!</h1>
{% endif %}
