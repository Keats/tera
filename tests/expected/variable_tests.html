<h1>Hello!</h1>

<h1>Hello again!</h1>
